crates/re_types/src/archetypes/arrows3d.rs

```diff
@@ -30,7 +30,7 @@ use ::re_types_core::{DeserializationError, DeserializationResult};
 /// use std::f32::consts::TAU;
 ///
 /// fn main() -> Result<(), Box<dyn std::error::Error>> {
-///     let (rec, storage) = rerun::RecordingStreamBuilder::new("rerun_example_arrow3d").memory()?;
```

Comment:
All of our examples now pass `None` as the `flush_timeout`. Reading the docs:

> `flush_timeout` is the minimum time the [`TcpSink`][`crate::log_sink::TcpSink`] will
> wait during a flush before potentially dropping data.  Note: Passing `None` here can cause a
> call to `flush` to block indefinitely if a connection cannot be established.

At the same time we have `re_sdk::default_flush_timeout()` which return `Some(2s)`

So we are effectively teaching users (via example) not to use the default. I think this is weird.

At the same time, the examples in the docs for `spawn` and `spawn_opts` DOES make use of this, so even our examples are inconsistent.

What do we want our examples to behave? What do we want our users to do?

Without having thought through the actual flushing consequences, I would suggest that `spawn()` takes no arguments so that we pick the same default for all our examples (and for our users!).

Reply:
`spawn` is effectively just a wrapper around `connect` so I want them to be consistent, and users definitely need to be able to customize this...

We teach users to use `re_sdk::default_flush_timeout()` anytime we mention `connect()`, I'll do the same for `spawn()` then.

Reply:
…or we use the same pattern of `connect()` and `connect_opts(…)`.

I really wish Rust has default arguments 😭

Reply:
> …or we use the same pattern of `connect()` and `connect_opts(…)`.

That could be quite good. I'll give it a shot in a follow up.

> I really wish Rust has default arguments 😭

:yes: